### PR TITLE
Adding MailChimpApiKey mock tests

### PR DESCRIPTION
**Pull request by eddynaka**
## Changes
- Making code coverage 100%

image

* [ ] `ReleaseHistory.md` updated for non-trivial changes
* [x] Added unit tests

### CODE REVIEW

File: Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs [hunk 1]
```diff
@@ -22,6 +22,8 @@ public class HttpMockHelper : DelegatingHandler
             new HttpResponseMessage(HttpStatusCode.OK);
         public static readonly HttpResponseMessage NotFoundResponse =
             new HttpResponseMessage(HttpStatusCode.NotFound);
+        public static readonly HttpResponseMessage BadRequestResponse =
+            new HttpResponseMessage(HttpStatusCode.BadRequest);
```
Comment (Bpendragon) [inline Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs:26]: Suggest also adding `Forbidden` and `InternalServerError` for use in other tests.

File: Src/Plugins/Tests.Security/Validators/SEC101_027.MailChimpApiKeyValidatorTests.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+using Newtonsoft.Json;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    /// <summary>
+    /// Testing SEC101/027.MailChimpApiKeyValidator
+    /// </summary>
+    public class MailChimpApiKeyValidatorTests
```
Comment (v-luisangell) [inline Src/Plugins/Tests.Security/Validators/SEC101_027.MailChimpApiKeyValidatorTests.cs:25]: Please consider adding [ExcludeFromCodeCoverage] attribute to all the test classes to take their code out of the Code Coverage computation.

**Review by v-luisangell — APPROVED** [review]
Looks good =)

File: Src/Plugins/Tests.Security/Validators/SEC101_027.MailChimpApiKeyValidatorTests.cs (hunk 2 above)
Comment (eddynaka) [reply Src/Plugins/Tests.Security/Validators/SEC101_027.MailChimpApiKeyValidatorTests.cs:25]: done

File: Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs (hunk 1 above)
Comment (eddynaka) [reply Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs:26]: done

**Pull request merged**